### DISCUSSION TITLE
Not working to me

### DISCUSSION THREAD

**Discussion by gborges1982** [swar/Swar-Chia-Plot-Manager#92] [Q&A]
Hi,

I followed all the steps using W10, setup de config and but nothing happens. 
1 - when I started the manager, the GUI of chia oppened, is this normal? 

I checked de log file and see this : 
C:\Users\Ploter\AppData\Local\chia-blockchain\app-1.1.3\resources\app.asar.unpacked\daemon
true
Error loading config - using defaults
startUrl file:///C:\Users\Ploter\AppData\Local\chia-blockchain\app-1.1.3\resources\app.asar\build\renderer\index.html
C:\Users\Ploter\AppData\Local\chia-blockchain\app-1.1.3\resources\app.asar.unpacked\daemon
true
C:\Users\Ploter\AppData\Local\chia-blockchain\app-1.1.3\resources\app.asar.unpacked\daemon
true
Running python executable: 
child process success
(node:5476) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Have cert
stderr: Socket error: connect, 10061

**Comment by heavensn** [comment]
me too
